### ISSUE TITLE
Youtube videos not downloading on windows version

### ISSUE THREAD

**Issue by stas1c** [persepolisdm/persepolis#955]
**System Details:**
* Operating System: Windows 11
* Persepolis Version: 4.0.1
* How do you install Persepolis? From installer here

Log File:
2024-03-23 09:16:08,176 - Persepolis - ERROR - Spider couldn't find file name

2024-03-23 09:16:10,209 - Persepolis - ERROR - Spider couldn't find file name

2024-03-23 09:16:17,361 - Persepolis - INFO - 8b266f0cc912c5c7 Starts

2024-03-23 09:16:18,381 - Persepolis - INFO - 4d2067626196c7ae Starts

2024-03-23 09:16:29,574 - Persepolis - ERROR - ffmpeg error: Not detecting m3u8/hls with non standard extension and non standard mime type

    Last message repeated 6 times

[mov,mp4,m4a,3gp,3g2,mj2 @ 00000215b6d22140] moov atom not found

[in#0 @ 00000215b6d0a6c0] Error opening input: Invalid data found when processing input

Error opening input file C:/Users/lonestar/AppData/Local/persepolis/MARKO HIETALA - Left On Mars (feat. Tarja Turunen) (OFFICIAL MUSIC VIDEO).mp4.

Error opening input files: Invalid data found when processing input

**Issue Description and steps to reproduce:**

**Comment by mostafaasadi** [comment]
Hi
would you please send me the link?

**Comment by stas1c** [comment]
Hi!
https://www.youtube.com/watch?v=1TgQOj3rwzE

**Comment by mostafaasadi** [comment]
@stas1c and did you change advanced options of video and audio formats? or with best quality?

**Comment by stas1c** [comment]
just tried to download with all the default settings.

**Comment by mostafaasadi** [comment]
so i can not see this error,  I test with two version of windows 10 and 11 and we don't have it, can you test again?

**Comment by stas1c** [comment]
I tried with other videos from this and other channels, and it's working fine. But particularly this doesn't download.  Tried to download this video with other app ([yt-dlg 1.8.5]) and there was no problem.

**Update:**
Changing advanced options to these helped - downloaded successfully.
![Screenshot 2024-03-24 102838](https://github.com/persepolisdm/persepolis/assets/43643488/b61b8171-8318-41d2-b64c-3f398e4a35b2)
